src/components/AudioDetails/Related.vue

```diff
@@ -3,22 +3,59 @@
     <h4 class="b-header mb-6">
       {{ $t('audio-details.related-audios') }}
     </h4>
-    <AudioTrack
-      v-for="audio in relatedAudios"
-      :key="audio.id"
-      :audio="audio"
-      :is-compact="true"
-      layout="row"
-      class="mb-12"
-    />
+    <template v-if="!$fetchState.error">
+      <AudioTrack
+        v-for="audio in audios"
+        :key="audio.id"
+        :audio="audio"
+        layout="row"
+        size="m"
+        class="mb-12"
+      />
+      <LoadingIcon v-show="$fetchState.pending" />
+    </template>
+    <p v-show="!!$fetchState.error">{{ errorMessage }}</p>
   </aside>
 </template>
 
 <script>
+import { ref } from '@nuxtjs/composition-api'
+import { AUDIO } from '~/constants/media'
+import useRelated from '~/composables/use-related'
+
 export default {
   name: 'RelatedAudios',
   props: {
-    relatedAudios: {},
+    audioId: {
+      type: String,
+      required: true,
+    },
+    service: {},
+  },
+  /**
+   * Fetches related audios on `audioId` change
+   * @param props
+   * @param {string} props.audioId
+   * @param {any} props.service
+   * @return {{audios: (Ref<UnwrapRef<[]>>|Ref<AudioDetail[]>)}}
```

Comment:
I find it helpful to add some extra spacing so these types don't run together and are more readable

```suggestion
   * @return {{ audios: (Ref<UnwrapRef<[]>> | Ref<AudioDetail[]>) }}
```

What does `Ref<UnwrapRef<[]>>` do? `UnwrapRef` sounds like something that should be passed a `Ref` itself, not passed _to_ a `Ref` but I'm uneducated on this at the moment.

Reply:
Thank you for spacing suggestions, this way JSDoc comments look consistent with the codebase.

As for `Ref<UnwrapRef<[]>>`, it was an auto-generated type annotation, and it's most likely incorrect, I will remove it. Thank you for spotting it!